src/RadioButtons/RadioButtons.js

```diff
@@ -1,7 +1,39 @@
 import "./RadioButtons.css";
+import Panel from "components/Panel/Panel";
+import Input from "components/Input/Input";
 
 const RadioButtons = () => {
-  return null;
+
+
+// reference: https://www.youtube.com/watch?v=GDh9uToZkYg&ab_channel=dcode
+
+  return (
+    <Panel>
+      <div className="radioSection">
+      <Input
+        label={'Apple '}
```

Comment:
```suggestion
        label='Apple '
```

You don't need to wrap in `{}` unless it is a variable